src/editor.tsx

```diff
@@ -31,6 +32,7 @@ class MonacoEditor extends React.Component<MonacoEditorProps> {
     editorDidMount: noop,
     editorWillMount: noop,
     onChange: noop,
+    selectChangedValue: true,
```

Comment:
I don't know whether this should be a global option. Please find out why we use `pushEditOperations` so we can decide whether it makes sense to have a global option or require users to just call `this.editor.setValue(value);` manually.